src/tools/r2rdump/CoreDisTools.cs

```diff
@@ -34,17 +40,36 @@ public enum TargetArch
         [DllImport("coredistools.dll")]
         public static extern void FinishDisasm(IntPtr Disasm);
 
-        public unsafe static string GetCodeBlock(IntPtr Disasm, int Address, int Offset, byte[] image, int Size)
+        public unsafe static string GetCodeBlock(IntPtr Disasm, RuntimeFunction rtf, int imageOffset, byte[] image)
         {
-            fixed (byte* p = image)
+            StringBuilder sb = new StringBuilder();
+
+            int rtfOffset = 0;
+            int codeOffset = rtf.CodeOffset;
+            Dictionary<int, GcInfo.GcTransition> transitions = rtf.Method.GcInfo.Transitions;
+            GcSlotTable slotTable = rtf.Method.GcInfo.SlotTable;
+            while (rtfOffset < rtf.Size)
             {
-                IntPtr ptr = (IntPtr)(p + Offset);
-                DumpCodeBlock(Disasm, (ulong)Address, ptr, Size);
+                int instrSize = 1;
+                fixed (byte* p = image)
+                {
+                    IntPtr ptr = (IntPtr)(p + imageOffset + rtfOffset);
+                    instrSize = DumpInstruction(Disasm, (ulong)(rtf.StartAddress + rtfOffset), ptr, rtf.Size);
+                }
+                IntPtr pBuffer = GetOutputBuffer();
+                string instr = Marshal.PtrToStringAnsi(pBuffer);
+
+                sb.Append(instr);
+                if (transitions.ContainsKey(codeOffset))
+                {
+                    sb.AppendLine($"\t\t\t\t{transitions[codeOffset].GetSlotState(slotTable)}");
```

Comment:
It seems we're mixing tab and space indentation. Could you make sure we're consistent here. Ie, in several places you are using `string tab = "    ";` which has 4 spaces instead of a tab.